src/auto-evo/selection_pressure/RootPressure.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace AutoEvo;
+
+public class RootPressure : SelectionPressure
+{
+    // Needed for translation extraction
+    // ReSharper disable ArrangeObjectCreationWhenTypeEvident
+    private static readonly LocalizedString NameString = new LocalizedString("MICHE_ROOT_PRESSURE");
+
+    // ReSharper restore ArrangeObjectCreationWhenTypeEvident
+
+    public RootPressure() : base(1, [
+        new RemoveOrganelle(_ => true),
+        new AddOrganelleAnywhere(_ => true),
+        new AddOrganelleAnywhere(organelle => organelle.InternalName == "nucleus"),
+    ])
+    {
+    }
+
+    public override LocalizedString Name => NameString;
+
+    public override float Score(Species species, SimulationCache cache)
+    {
+        return 1;
+    }
+
+    public override float GetEnergy()
+    {
+        return 0;
+    }
+
+    public override LocalizedString GetDescription()
+    {
+        return Name;
```

Comment:
A default implementation of this can be in the base class, with only child classes that want to override `return Name;` needing to then define this method explicitly.